algebra/src/to_field_vec.rs

```diff
@@ -6,9 +8,10 @@ use crate::{
         ProjectiveCurve,
     },
     Fp2, Fp2Parameters, FpParameters, Field, PrimeField,
+    serialize::SerializationError,
 };
 
-type Error = Box<dyn std::error::Error>;
+type Error = Box<SerializationError>;
```

Comment:
Sure. I didn't remove the `Box`, cause I thought that it may be here for performance reasons (e.g. to avoid allocating space on stack for a big `Error`), but I don't think it is a case here

Reply:
Yeah, earlier I didn't want to do the work of defining a custom Error enum, so I took the lazy shortcut =P

Reply:
done :)